### PR TITLE
[12.0][ADD] agreement_helpdesk_mgmt

### PR DESCRIPTION
**Pull request by marcelsavegnago**
Based on https://github.com/ursais/osi-addons/tree/12.0/agreement_helpdesk

cc @bodedra @smangukiya

### CODE REVIEW

File: agreement_helpdesk_mgmt/README.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    :target: https://translation.odoo-community.org/projects/contract-12-0/contract-12-0-agreement_helpdesk_mgmt
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/110/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Odoo Agreement App does not provide an easy way to access helpdesk tickets
+related to an agreement. Some organizations needs to have a quick access to
```
Comment (eduaparicio) [inline agreement_helpdesk_mgmt/README.rst]:
```suggestion
related to an agreement. Some organizations need to have a quick access to
```

File: agreement_helpdesk_mgmt/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Odoo Agreement App does not provide an easy way to access helpdesk tickets
+related to an agreement. Some organizations needs to have a quick access to
```
Comment (eduaparicio) [inline agreement_helpdesk_mgmt/readme/DESCRIPTION.rst]:
```suggestion
related to an agreement. Some organizations need to have a quick access to
```

**Review by eduaparicio — APPROVED** [review]
Minor text suggestions. Approved!

**Review by jpcarassato — APPROVED** [review]
Ok!

**Comment by marcelsavegnago** [comment]
@bodedra @smangukiya can you review this PR please?

**Comment by marcelsavegnago** [comment]
@max3903 Can you review this PR please? I used one of the Ursais modules as a reference and added Escodoo as a secondary author but I have no problem removing it.

**Comment by marcelsavegnago** [comment]
@dreispt can you review please ?

**Comment by marcelsavegnago** [comment]
ping @bodedra

File: agreement_helpdesk_mgmt/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    'website': 'https://github.com/oca/contract',
+    'images': ['static/description/banner.png'],
+    'depends': [
+        "helpdesk_mgmt",
+        "agreement_serviceprofile",
+    ],
+    'data': [
+        'views/helpdesk_ticket.xml',
+        'views/agreement.xml',
+    ],
```
Comment (bodedra) [inline agreement_helpdesk_mgmt/__manifest__.py:20]:
```suggestion
    ],
    'maintainers': [
        'bodedra'
    ],
```

**Review by bodedra — COMMENTED** [review]
Would you please add me as maintainer.

I have restarted runbot for functional review.

**Comment by bodedra** [comment]
> @max3903 Can you review this PR please? I used one of the Ursais modules as a reference and added Escodoo as a secondary author but I have no problem removing it.

@marcelsavegnago Yes please. Just add yourself as a Contributor.

File: agreement_helpdesk_mgmt/readme/CREDITS.rst [hunk 4]
```diff
@@ -0,0 +1,4 @@
+The development of this module has been financially supported by:
+
+* Open Source Integrators
+* `Escodoo <https://www.escodoo.com.br>`_
```
Comment (bodedra) [inline agreement_helpdesk_mgmt/readme/CREDITS.rst]:
```suggestion
```

**Review by bodedra — CHANGES_REQUESTED** [review]

**Comment by bodedra** [comment]
@marcelsavegnago can you please squash commits into one before proceeding to merge?

**Review by bodedra — APPROVED** [review]
Functional reviewed. :+1: 

/ocabot merge nobump

**Comment by bodedra** [comment]
@sbejaoui @ygol @pedrocasi
Would you please merge this PR?

**Pull request merged**